gwswlib/threedi.py

```diff
@@ -64,6 +73,31 @@ def import_hydx(self, hydx):
                 "Connection",
             )
 
+            linkedprofile = None
+            if connection.typeverbinding in ["GSL", "OPL", "ITR", "DRL"]:
```

Comment:
In hoeverre is het ook mogelijk/handig om een `if .... not in ....: break` te doen om naar de volgende in het lijstje connection nodes over te stappen? Scheelt weer een niveau inspringen.

Reply:
bij dit snap ik niet wat je bedoeld

Reply:
Je code ziet er nu zo uit:

    for connection in connections:
        if connection.typeverbinding in ["GSL", "OPL", "ITR", "DRL"]:
            doe_het_echte_werk()  # Het echte werk is flink ingesprongen.

Wat ook kan:

    for connection in connections:
        if connection.typeverbinding not in ["GSL", "OPL", "ITR", "DRL"]:
            break  # Breekt de huidige 'connection' af en gaat naar de volgende connection in de 'for' loop.
        doe_het_echte_werk()  # Minder ver ingesprongen.

De tweede versie is misschien wat explicieter.

Waar het me om gaat: beginnende python programmeurs gebruiken vaak geen `break` of `return` om een functie of loop af te breken. Ze doen het met if/else.
Vaak wordt het door return/break wat explicieter/duidelijker wat je bedoeling is.

Reply:
Ik snap hem. Deze kan hier echter niet, omdat het tweede if statement er op verder bouwt. Als hij de for loop zou afbreken dan zouden geen PMP of OVS'en worden gemaakt